next/src/components/global/Footer/Footer.module.scss

```diff
@@ -0,0 +1,63 @@
+.footer {
+  background-color: var(--neutral-800, #070101);
+  padding: clamp(16px, calc(40vw / 7.68), 40px) 0 clamp(56px, calc(80px / 7.68), 80px);
+  display: grid;
+  row-gap: clamp(32px, calc(128vw / 7.68), 128px);
```

Comment:
Be careful with that types of clamp values, because the 1st one (32px) is much lower than the 2nd one (128), and it may happen that, the lowest value (`32px`) can be never achieved.